massa-pos-exports/src/cycle_info.rs

```diff
@@ -61,18 +61,22 @@ impl CycleInfoHashComputer {
         Hash::compute_from(&buffer)
     }
 
-    fn compute_roll_entry_hash(&self, address: &Address, roll_count: u64) -> Hash {
+    fn compute_roll_entry_hash(
+        &self,
+        address: &Address,
+        roll_count: u64,
+    ) -> HashXof<HASH_XOF_SIZE_BYTES> {
```

Comment:
I think we should either add a comment and make it clear that this hash is never included in the db hash. I think we may be able to remove entirely the Hash tracker from the cycle info struct.
I'm not sure if it's the same with deferred credits as they are used speculatively, but I think we can remove it also.

Reply:
hashes should not be computed at speculative time, only when finalizing, so yes we should be able to remove all hash trackers except the automatic one in the db

Reply:
done